tests/testthat/test-shinytest2-tm_g_bivariate.R

```diff
@@ -0,0 +1,79 @@
+testthat::test_that("e2e - tm_g_bivariate: module is initialised with the specified defaults", {
+  skip_if_too_deep(5)
+
+  # needed untill this is merged https://github.com/insightsengineering/teal/pull/1198
+  require(shinytest2)
+
+  data <- teal_data()
+  data <- within(data, {
+    require(nestcolor)
+    CO2 <- data.frame(CO2)
+  })
+  datanames(data) <- c("CO2")
+
+  app <- TealAppDriver$new(
+    data = data,
+    modules = modules(
```

Comment:
is there a need for modules, if there is one module?
```suggestion
    modules = 
```

Reply:
True, not needed. I'll remove it.